packages/bitcore-node/src/modules/firocoin/api/firo-routes.ts

```diff
@@ -24,9 +29,178 @@ FiroRoutes.get('/api/:chain/:network/contract/:contractAddress', async (req, res
 FiroRoutes.get('/api/:chain/:network/prices', async (_, res) => {
   try {
     res.json({
-      NXC: 0.33 * 1e9 // price in usd * 1e9
+      NXC: 0.33 * 1e9, // price in usd * 1e9
     });
   } catch (err) {
     res.status(500).send(err);
   }
 });
+
+FiroRoutes.get('/api/:chain/:network/token', async (req, res) => {
+  let { chain, network, paging } = req.params;
+  try {
+    const limit = 20;
```

Comment:
We should receive limit from params and setting default

Reply:
ok kub